test/types/index.ts

```diff
@@ -251,3 +251,26 @@ server.setSchemaCompiler(function (schema: object) {
 server.addSchema({})
 
 server.printRoutes()
+
+server.ready(function (err) {
+  if (err) throw err
+})
+
+server.ready(function (err, done) {
+  if (err) throw err
```

Comment:
Should this if statement and throw be removed if using the callback?

Reply:
There are not suggested paths on how handle this error, it's a decision that belongs to the user.
For the sake of the test, throw error is ok IMHO.

Reply:
if there is a done callback, the error should be passed there.

Reply:
and not thrown